config/diagnostics/ocean3d/config_ocean_drift.yaml

```
@@ -0,0 +1,22 @@
+datasets:
+  - catalog: ci
```

please leave `catalog: null` in the final version of the PR
```suggestion
  - catalog: null
```